lms/djangoapps/teams/views.py

```diff
@@ -889,6 +888,13 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
+        if course_module.teams_max_size and team.users.count() >= course_module.teams_max_size:
+            return Response(
```

Comment:
Oh, right, course_module.teams_max_size may return None so you can't just remove the check. But I don't like using the "falsey" check you have because 0 will also short-circuit the check.

How about course_module.teams_max_size is not None and team.users.count() >= course_module.teams_max_size?